src/integrations/pf2e-integration.ts

```diff
@@ -183,6 +183,28 @@ export class PF2eIntegration {
   }
 }
 
+// Hook to provide world creation timestamp to core S&S when requested
+Hooks.on(
+  'seasons-stars:pf2e:getWorldCreationTimestamp',
+  (timestampData: { worldCreationTimestamp: number | undefined }) => {
+    try {
+      const worldCreatedOn = (game as any).pf2e?.settings?.worldClock?.worldCreatedOn;
```

Comment:
This isn't the only place `worldCreatedOn` is accessed in this file. We should consider consolidating them. Maybe there's a better way than this hook for exposing the value to the overall module.